x/concentrated-liquidity/msg_server.go

```diff
@@ -6,21 +6,22 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	clmodel "github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/model"
 	"github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/types"
 )
 
 type msgServer struct {
 	keeper *Keeper
 }
 
-func NewMsgServerImpl(keeper *Keeper) types.MsgServer {
+func NewMsgServerImpl(keeper *Keeper) clmodel.MsgServer {
 	return &msgServer{
 		keeper: keeper,
 	}
 }
 
 // TODO: tests, including events
-func (server msgServer) CreatePosition(goCtx context.Context, msg *types.MsgCreatePosition) (*types.MsgCreatePositionResponse, error) {
+func (server msgServer) CreatePosition(goCtx context.Context, msg *clmodel.MsgCreatePosition) (*clmodel.MsgCreatePositionResponse, error) {
```

Comment:
Can we keep this message in `types`? Only `MsgCreatePool` should be in the `model` package since it's the only one related to the pool model

Reply:
This is alligned with gamm module where we only keep `MsgCreatePool` definitions in the model packages